src/js/components/Button/StyledButtonKind.js

```diff
@@ -226,6 +227,12 @@ const StyledButtonKind = styled.button`
   ${props => props.plain && plainStyle(props)}
   ${props => !props.plain && basicStyle(props)}
   ${props => !props.plain && kindStyle(props)}
+  ${props =>
+    !props.plain &&
+    props.align &&
+    `
+  text-align: ${props.align};
```

Comment:
Since `align` underlying is resolved to `text-align` I'm thinking we might want to communicate better the name of the prop as `textAlign` rater than `align`.

Reply:
So, the trick here is that when the Button contains only text, we need `text-align` but when there is an icon and a label, we actually set `justify` on the Box that contains them. So, `align` here is more about the visual manifestation and less about the underlying implementation, which varies based on context.